CHANGELOG.md

```diff
@@ -1,18 +1,29 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.1.2](https://github.com/mkdocstrings/handler-template/releases/tag/0.1.2) - 2023-01-06
+
+<small>[Compare with 0.1.1](https://github.com/mkdocstrings/handler-template/compare/0.1.1...0.1.2)</small>
+
+### Code Refactoring
+
+- Added doc strings to collect/render to help a user implement their custom feature set.
+
```

Comment:
Thanks, but you don't have to update the changelog manually, I have an automated action for that :smile: 
Can you simply revert changes in CHANGELOG.md?

Reply:
Done, yep.  I realized that when I moved on to the mkdocstrings PR.